crates/fonticulus/src/basictables.rs

```diff
@@ -164,6 +165,15 @@ pub fn compile_os2(
         .open_type_os2_subscript_x_size
         .unwrap_or((upm * 0.65).round() as i32) as i16;
 
+    let unicodes:HashSet<u32> = mapping.keys().cloned().collect();
```

Comment:
I'm going to move this function so it becomes a method of the os2 table since it makes more sense for the code to live there. I may just end up making the method signature take the cmap table as input but I'll think about it.